src/components/TopBar.tsx

```diff
@@ -106,15 +111,26 @@ const ButtonWrapper = styled('div')({
   justifyContent: 'flex-end',
   textAlign: 'right',
   width: '20%',
+  paddingTop: 4,
+  paddingLeft: 10,
+  '& > *:last-of-type': {
+    marginRight: 26,
+  },
   '@media (max-width: 850px)': {
     width: '33%',
   },
   '@media (max-width: 600px)': {
-    width: '50%',
+    width: '45%',
+    paddingRight: 20,
+    '& > *:last-of-type': {
+      marginRight: 10,
+    },
+    '& > *:last-child': {
+      marginRight: 0,
+    },
```

Comment:
this seems a bit contradictory to me

Reply:
The CSS selector is indeed a bit weirdly written. It's probably hard to write it better though given that the button-property is abstract enough that it takes any element.

It will also break when a third button is introduced into the mix.

Given the tight deadline on this feature, we're going ahead and merge it anyways.